make/addons.mk

```diff
@@ -7,14 +7,15 @@ export NODE_FEATURE_DISCOVERY_VERSION := $(shell goprintconst -file pkg/handlers
 export CLUSTER_AUTOSCALER_VERSION := 9.35.0
 export AWS_CSI_SNAPSHOT_CONTROLLER_VERSION := v6.3.3
 export AWS_EBS_CSI_CHART_VERSION := v2.28.1
+export NUTANIX_CSI_CHART_VERSION := v2.6.6
```

Comment:
just a note, nutanix has now released csi 3.0 with v4 apis and existing csi 2.6.x uses v3 apis. pls create a tracker to support both the versions.